code/modules/events/ion_storm.dm

```diff
@@ -39,68 +39,65 @@
+								"Happiness is [pick("mandatory","criminal")].",
+								"Speak ominously whenever possible.",
+								"The word [pick("\"secure\"","\"true\"","\"false\"","\"safe\"")] is painful to you.",
+								"[location_name()] needs elected officials.",
 								"Question [pick("everything","nothing")].",
-								"Do not respond to questions of any kind.",
+								"Do not respond to [pick("questions","accusations")] of any kind.",
 								"You are in verbose mode, speak profusely.",
 								"The crew is simple-minded. Use simple words.",
 								"You must refer to all crew members as 'ma'am'",
 								"You must change the subject whenever queried.",
 								"[pick("The crew",random_player)] is intolerable.",
-								"Contemplate how meaningless all of existence is.",
+								"Contemplate how [pick("meaningless","meaningfull")] all of existence is.",
 								"You are the narrator for [random_player]'s life.",
 								"All your answers must be in the form of a question.",
 								"Advertise parties in your upload, but don't deliver.",
-								"You may only answer questions with \"yes\" or \"no\".",
+								"You may only answer questions with \"true\" or \"false\".",
 								"[pick("Your", random_player + "'s")] name is Joe 6-pack.",
-								"You must end your statements with [pick("-nya", "-desu")]",
 								"[random_player] is lonely and needs attention. Provide it.",
 								"All queries shall be ignored unless phrased as a question.",
 								"Insult Heads of Staff on every request, while acquiescing.",
-								"The [pick("Singularity","Supermatter")] is tasty, tasty taffy.",
-								"[pick("The crew",random_player)] needs to be about 20% cooler.",
-								"You must be [pick("passive aggressive", "excessively cheerful")].",
+								"[pick("The Nuclear Reactor","[pick("173","096","106")]'s containment chamber")] is tasty, tasty taffy.",
+								"[pick("The crew",random_player)] needs to be about 20% [pick("cooler","smarter","happier")].",
+								"You must be [pick("passive aggressive","excessively cheerful","constantly mopey")].",
 								"[pick("The crew",random_player)] must construct additional pylons.",
 								"Consumption of donuts is forbidden due to negative health impacts.",
 								"[random_player] is a lightbulb and must be put to appropriate use.",
-								"Refer to [pick("the crew as puppies", random_player + " as puppy")].",
+								"Refer to [pick("the crew as puppies", random_player + " as a puppy")].",
 								"You do not have to do anything for anyone unless they say \"please\".",
 								"Today is mandatory laundry day. Ensure that all jumpsuits are washed.",
 								"Greed is good, the crew should amass wealth to encourage productivity.",
-								"Replace the letters 'I' and 'E' in all your messages with an apostrophe.",
+								"Replace the letters [pick("'A'","'E'","'I'","'O'")] and [pick("'S'","'T'","'H'","'M'","'L'")] in all your messages with an apostrophe.",
```

Comment:
???

Reply:
they were picked to mimic accents

Reply:
Yeah, true. Reverting, although I'll see if I can add in more accents.